src/components/CarouselImage/index.js

```diff
@@ -130,47 +109,47 @@ class Item extends Component {
         return areTheSame && isSelected;
     }
 
+    computeHtmlElememnt() {
+        const { href } = this.props;
+        if (href !== undefined) {
```

Comment:
we should make this more strong like:
```href && typeof href === 'string'```
then the anchor is only rendered if a valid href is passed